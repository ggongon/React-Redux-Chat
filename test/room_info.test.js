import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import RoomInfo from '../js/components/room_info';

const currentRoom =  {
  name: 'Tea Chats',
  id: 0,
  users: [ 'Ryan', 'Nick' ]
}

it('should render properly', () => {
  const roominfo = renderer.create(<RoomInfo currentRoom={currentRoom} />)
  expect(roominfo).toMatchSnapshot();
});
