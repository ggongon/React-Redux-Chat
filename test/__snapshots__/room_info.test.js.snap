// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<div
  className="roomInfo center"
>
  <h2>
    Tea Chats
  </h2>
  <h5>
    Ryan, Nick
  </h5>
</div>
`;
